react/game/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { reduxForm } from 'redux-form';
+import ROUTES from '@constants/routes';
+import { Link } from 'react-router-dom';
+import { func } from 'prop-types';
+import Logo from '@components/Logo';
+
+import styles from './styles.scss';
+
+class LoginForm extends Component {
+  handleSubmit = event => {
+    event.preventDefault();
+    this.props.onSubmit(event);
+  };
+
+  render() {
+    return (
+      <form className={styles.loginForm} onSubmit={this.handleSubmit}>
+        <Logo />
+        <input type="text" placeholder="username" />
+        <input type="password" placeholder="password" />
+        <button type="submit">Login</button>
+        <p className={styles.message}>
+          Not registered? <Link to={ROUTES.REGISTER.path}>Create an account</Link>
+        </p>
+      </form>
+    );
+  }
+}
+
+LoginForm.propTypes = {
+  onSubmit: func.isRequired
+};
+
+export default reduxForm({
+  form: 'loginForm' // a unique identifier for this form
```

Comment:
Extract this string to a const file, in `src/constants/formNames.js`. Think that a form can handle many screens, like a wizard form, and also sometimes u need to read data from that form and for that u need the form name

Reply:
Great contribution, thanks 👌🏼